specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -81,9 +81,14 @@ Link to the official Windows App SDK timeline can be found
 
 The code in Main would follow the pattern below:
 
--   The Registration will take in a CLSID of the COM server as part of the Activator setup. The
-    registration API will simply abstract away the COM details from the developer including
-    registration of the in-process Windows App SDK component.
+-   All apps will call into the registration API.
```

Comment:
Editorial note - consider a stronger form of statements like this, basically eliminating the word "will":

> Apps start by calling into the registration API

Similarly below:

> For packaged applications, registration with the notification services is handled automatically.

And:

> Use the `CoreApp` activation event system to determine the `ActivationKind` - for `PushNotification`, the activation arguments object is an instance of `PushActivatedEventArgs`.

... no change required for now.